### ISSUE TITLE
WiiU Pro Controller controls mouse cursor

### ISSUE THREAD

**Issue by Vash63** [ValveSoftware/SteamOS#461]
With the default configuration, pairing a WiiU Pro Controller causes the left joystick to function both as a joystick and a mouse simultaneously. I was able to fix this by adding a match rule to an Xorg.conf as below, but it would be great if this was upstreamed or a better fix added to SteamOS so that the controller would work 'out of the box'.

Stolen Xorg.conf file from [Ubuntu Wiki](https://help.ubuntu.com/community/Sixaxis)
``` 
Section "InputClass"
        Identifier "joystick catchall"
        MatchIsJoystick "on"
        MatchDevicePath "/dev/input/event*"
        Driver "joystick"
        Option "StartKeysEnabled" "False"       #Disable mouse
        Option "StartMouseEnabled" "False"      #support
EndSection
```

**Comment by Plagman** [comment]
We'll look into this to make sure this controller works well out of the box, thanks.

**Comment by mmolignano** [comment]
@Vash63 What do you mean by "default configuration"? How are you pairing the controller with steamos currently?

**Comment by mmolignano** [comment]
@Vash63 I don't seem to have an Xorg.conf file by default (at least not in /etc/X11). Did you just create a new one and only add the configuration you posted above to it?

**Comment by Tele42** [comment]
@mmolignano yes, any X config section that is not in xorg.conf is autoconfigured on start of X, making it unneeded in many scenarios.

**Comment by stepotronic** [comment]
I would like to add that for me the installation of xserver-xorg-input-xwiimote triggered it. Removing it solved the issue. It was also not neccessary to install it for wiiu pro controllers as some tutorials suggested.
